src/components/Board.vue

```diff
@@ -0,0 +1,472 @@
+    }
+    if (grid != null) {
+      grid.style.width = `${this.width * 40}px`;
+      await this.clearHelper();
+      await this.createBoard();
+      await this.fillCells();
+      this.stopTimer();
+      this.resetTimer();
+      this.timerEnabled = true;
+    }
+  }
+
+  private createBoard(): void {
+    const bombsArray = Array(this.$store.state.bombsAmount).fill("bomb");
+    const emptyArray = Array(
+      this.width * this.width - this.$store.state.bombsAmount
+    ).fill("empty");
+    const gameArray = emptyArray.concat(bombsArray);
+    const shuffledArray = gameArray.sort(() => Math.random() - 0.5);
+    this.squares = [...shuffledArray];
+  }
+
+  private fillCells(): void {
+    const elements = document.getElementsByClassName("square");
+    for (let i = 0; i < elements.length; i++) {
+      this.cells.push(elements[i]);
+    }
+    this.addNumbers();
+  }
+
+  public clicked(square) {
+    if (this.timerEnabled === true) {
+      this.resetTimer();
+      this.startTimer();
+      this.timerEnabled = false;
+    }
+    if (square === undefined) {
+      square = window.event!.target;
+    }
+    const currentId = square.id;
+    if (this.isGameOver) return;
+    if (
+      square.classList.contains("checked") ||
+      square.classList.contains("flag")
+    )
+      return;
+    if (square.classList.contains("bomb")) this.gameOver();
+    const total = square.getAttribute("data");
+    if (total > 0 && total <= this.classesToDelete.length) {
+      square.classList.add("checked");
+      square.classList.add(this.classesToDelete[total - 1]);
+      square.innerHTML = total;
+      if (total > 3) {
+        this.gameState = States.SoClose;
+      }
+      return;
+    }
+    if (!square.classList.contains("bomb")) {
+      this.gameState = States.NiceTry;
+    }
+    this.checkSquare(currentId);
+    square.classList.add("checked");
+  }
+
+  private addFlag(): void {
+    const square = window.event!.target as HTMLTextAreaElement;
+    if (this.isGameOver) return;
+    if (
+      !square.classList.contains("checked") &&
+      this.$store.state.flags <= this.$store.state.bombsAmount
+    ) {
+      if (!square.classList.contains("flag")) {
+        square.classList.add("flag");
+        square.innerHTML = "🚩";
+        this.$store.dispatch("updateFlags", this.flags + 1);
+        this.checkForWin();
+      } else {
+        square.classList.remove("flag");
+        square.innerHTML = "";
+        this.$store.dispatch("updateFlags", this.flags - 1);
+      }
+    }
+  }
+
+  private addNumbers(): void {
+    for (let i = 0; i < this.cells.length; i++) {
+      let total = 0;
+      const isLeftEdge = i % this.width === 0;
+      const isRightEdge = i % this.width === this.width - 1;
+      if (this.cells[i].classList.contains("empty")) {
+        if (
+          i > 0 &&
+          !isLeftEdge &&
+          this.cells[i - 1].classList.contains("bomb")
+        )
+          total++;
```

Comment:
przenies  warurnek do jednej linii użyj nawiasów klamrowych za nawiasem zamykajacym if czyli `if (...) {...}`.

Reply:
to akurat prettier mi tak pozamieniał, ale ok zmienię